contract/contracts/auditLog.sol

```diff
@@ -1,39 +1,51 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.5.x;
 
-/**
- * @title Storage
- * @dev Store & retrieve value in a variable
- */
-contract auditLog {
-    // The table for mapping DB table name and address which save the the audit log of DB
-    // operations.
-    mapping(string => address) auditLogMap;
+contract SK2AddrTable {
```

Comment:
Why `SK2AddrTable`?

Reply:
SK means secret key. I have mentioned it in the `authRole.sol`